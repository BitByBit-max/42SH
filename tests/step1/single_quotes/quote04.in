echo 'hello big 'string' is here'
