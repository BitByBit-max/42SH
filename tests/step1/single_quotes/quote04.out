hello big string is here
